packages/extension-chakra-storefront/src/pages/product-list/index.jsx

```diff
@@ -43,7 +43,7 @@ import ProductTile, {Skeleton as ProductTileSkeleton} from '../../components/pro
 import {HideOnDesktop} from '../../components/responsive'
 // TODO: Refinements components will be migrated in separate PR
 // import Refinements from '../../pages/product-list/partials/refinements'
-import CategoryLinks from '../../pages/product-list/partials/category-links'
+// import CategoryLinks from '../../pages/product-list/partials/category-links'
```

Comment:
Comment out to avoid the complilation error, we have a ticket to tackle this component